### PR TITLE
SAK-46654 resolve several GBNG issues via two reversions

### PR DESCRIPTION
**Pull request by ottenhoff**
This reverts SAK-43022 and SAK-45531 to resolve several GBNG issues:

a) Fixes course grade overrides when using Percentage grading

b) Fixes incorrect points display in Course Grade Override Panel when using "show points"

c) Fixes incorrect points display when using show points on the Course Grade column 

d) Fixes incorrect calculated grade when using custom export with "show points" enabled and selected points total for grade

### CODE REVIEW

File: config/configuration/bundles/src/bundle/org/sakaiproject/config/bundle/default.sakai.properties [hunk 1]
```diff
@@ -1921,10 +1921,6 @@
 # Default: false
 # gradebook.coursegrade.displayed=true
 
-# Default for course grade calculated
-# Default: true
-# gradebook.coursegrade.showCalculatedGrade=false
```
Comment (jesusmmp) [inline config/configuration/bundles/src/bundle/org/sakaiproject/config/bundle/default.sakai.properties:1926]: Please, don't remove this property.

Comment (ottenhoff) [reply config/configuration/bundles/src/bundle/org/sakaiproject/config/bundle/default.sakai.properties:1926]: This PR reverts the two JIRAs. Please provide some feedback on the issues identified and how they can be *quickly* fixed. If they can't be fixed quickly, the two commits should be reverted until they can be fixed.

**Comment by jesusmmp** [comment]
@ottenhoff I've talked with Earle and Jorge about this problem. 
I'm agree with reverting it because the implementation isn't good because we have changed pointsEarned attribute even when the grade is not override. 

We really need that property, so I'm going to create a new jira with a new implementation asap. 
We would like that the new PR will be merged to 22 too.

Thanks @ern !

**Review by jesusmmp — APPROVED** [review]
LGTM!

**Review by ern — APPROVED** [review]

**Pull request merged**